dotnet/devops/android-cargo-config

```diff
@@ -0,0 +1,11 @@
+[target.aarch64-linux-android]
```

Comment:
This is now in the root of the .cargo folder in the repo

Reply:
For some reason, this doesn't work with the current Azure Pipeline images. I have to explicitly remove it, and copy it to the home folder, for it to work. This is a dependency for `Build.ps1`. Once we migrate over, we can remove all of these artifacts.